content/concepts/stream-multiplexing.md

```diff
@@ -58,23 +58,15 @@ yamux is currently supported in go and rust:
 
 [QUIC][wiki-quic] is a [transport](/concepts/transport/) protocol that contains a "native" stream multiplexer. libp2p will automatically use the native multiplexer for streams using a quic transport.
 
-quic is currently supported in go via [go-libp2p-quic-transport](https://github.com/libp2p/go-libp2p-quic-transport).
-
-#### spdy
-
-[SPDY][wiki-spdy] is a Google-developed protocol that was the precursor to HTTP/2. SPDY implements a stream multiplexer, which is supported by some libp2p implementations:
-
-- Go: [go-smux-spdystream](https://github.com/whyrusleeping/go-smux-spdystream)
-- Javascript: [js-libp2p-spdy](https://github.com/libp2p/js-libp2p-spdy)
+quic is currently supported in go via [go-libp2p-quic-transport](https://github.com/libp2p/go-libp2p/tree/master/p2p/transport/quic).
 
 #### muxado
 
 [muxado](https://github.com/inconshreveable/muxado) is a go stream muxing library, supported by go-libp2p via [go-smux-muxado](https://github.com/whyrusleeping/go-smux-muxado).
```

Comment:
Side note, is this still relevant @marten-seemann, @MarcoPolo and @julian88110?

Reply:
@mxinden removed here and on the implementations

Reply:
Thanks @DannyS03.